kill-counter.lic

```diff
@@ -0,0 +1,99 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#kill-counter
+=end
+
+unless HAVE_GTK
```

Comment:
i used performance-monitor as a template. However, based upon the use case it is desired to be something that could be called by command line and then send a signal to get whatever the current count is.